eng/pipelines/aggregate-reports.yml

```diff
@@ -94,13 +93,29 @@ jobs:
         Copy-Item data.js $(Build.ArtifactStagingDirectory)/staging
       displayName: 'Copy reports to artifact staging'
 
+    - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@3
+      displayName: 'Run CredScan'
+      inputs:
+        suppressionsFile: 'eng\CredScanSuppression.json'
+      condition: succeededOrFailed()
+    - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
+      displayName: 'Publish Security Analysis Logs'
+    - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
+      displayName: 'Post Analysis'
+      inputs:
+        GdnBreakAllTools: false
+        GdnBreakGdnToolCredScan: true
+        GdnBreakGdnToolCredScanSeverity: Error
+        GdnBreakBaselineFiles: $(Build.SourcesDirectory)/eng/java.gdnbaselines
+        GdnBreakBaselines: baseline      
+      condition: succeededOrFailed()
     # Run link verification at the end as it will run no matter the pipeline passing or failing.
     # If this task fails with an error it shouldn't prevent aggregate reports from being generated.
     - template: ../common/pipelines/templates/steps/verify-links.yml
       parameters:
         Directory: ""
         CheckLinkGuidance: $true
-
+        RunOnFailure: $true
```

Comment:
If this remains a bool we shouldn't need it to be `$` because that is only interesting for PS.

Reply:
Do you mean we can use `RunOnFailure: true` in the yaml script?

Reply:
The problem goes away as the tool PR got changed.